lib/weapons.json

```
@@ -2944,6 +2944,58 @@
       "Melee": 1
     }
   },
+  {
+    "id": "coherentsubatomicneedle",
+    "name": "Coherent Subatomic Needle",
+    "mount": "Main",
+    "type": "Melee",
+    "damage": [
+      {
+        "type": "kinetic",
+        "val": "1d6"
+      }
+    ],
+    "range": [
+      {
+        "type": "Threat",
+        "val": 2
+      }
+    ],
+    "tags": [],
+    "source": "SSC",
+    "license": "WHITE WITCH",
+    "license_level": 1,
+    "effect": "1/round you may force a character struck by this weapon must make a hull save. On a failed save, both you and the target are immobilized. At the end of each of its turns, the target can repeat the save, freeing both of you on success. As a quick action on your turn, you can break this immobilization effect on both of you to whip your target away, knocking them back 4 spaces in any direction. Otherwise you both remain immobilized.",
+    "description": "“To say we can pull this needle from ‘thin air’ is the best-fit phrase, but it’s not accurate. Not entirely. There’s no such thing as thin air, not in the way one means it when they use that phrase. ‘Agreeable Atomic Space’ — that’s what ‘thin air’ really is. A place where one could — if one could — coalesce utility from useless particulate mass with a thought. With a snap of our finger. Here, watch this —“",
+    "data_type": "weapon",
+    "aptitude": {}
+  },
+  {
+    "id": "retortloop",
+    "name": "Retort Loop",
+    "mount": "Heavy",
+    "type": "Cannon",
+    "damage": [
+      {
+        "type": "kinetic",
+        "val": "[armor]"
```

```suggestion
        "val": "3+1d6/Charge"
```
This might be inconsistent with other weapons, which put fixed damage *after* the variable damage.